docs/getting-started-guides/kubeadm.md

```diff
@@ -54,155 +65,209 @@ If you are not constrained, there are some other tools built to give you complet
+* SSH into the machine and become root if you are not already (for example,
+  run `sudo su -`).
 * If the machine is running Ubuntu or HypriotOS, run:
 
-      apt-get update && apt-get install -y apt-transport-https
-      curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
-      cat <<EOF > /etc/apt/sources.list.d/kubernetes.list
-      deb http://apt.kubernetes.io/ kubernetes-xenial main
-      EOF
-      apt-get update
-      # Install docker if you don't have it already.
-      apt-get install -y docker.io
-      apt-get install -y kubelet kubeadm kubectl kubernetes-cni
-
-   If the machine is running CentOS, run:
-
-      cat <<EOF > /etc/yum.repos.d/kubernetes.repo
-      [kubernetes]
-      name=Kubernetes
-      baseurl=http://yum.kubernetes.io/repos/kubernetes-el7-x86_64
-      enabled=1
-      gpgcheck=1
-      repo_gpgcheck=1
-      gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
-             https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
-      EOF
-      setenforce 0
-      yum install -y docker kubelet kubeadm kubectl kubernetes-cni
-      systemctl enable docker && systemctl start docker
-      systemctl enable kubelet && systemctl start kubelet
-
-The kubelet is now restarting every few seconds, as it waits in a crashloop for `kubeadm` to tell it what to do.
-
-Note: Disabling SELinux by running `setenforce 0` is required in order to allow containers to access the host filesystem, which is required by pod networks for example. You have to do this until kubelet can handle SELinux better.
+  ``` bash
+  apt-get update && apt-get install -y apt-transport-https
+  curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+  cat <<EOF > /etc/apt/sources.list.d/kubernetes.list
+  deb http://apt.kubernetes.io/ kubernetes-xenial main
+  EOF
+  apt-get update
+  # Install docker if you don't have it already.
+  apt-get install -y docker.io
+  apt-get install -y kubelet kubeadm kubectl kubernetes-cni
+  ```
+
+  If the machine is running CentOS, run:
+
+  ``` bash
+  cat <<EOF > /etc/yum.repos.d/kubernetes.repo
+  [kubernetes]
+  name=Kubernetes
+  baseurl=http://yum.kubernetes.io/repos/kubernetes-el7-x86_64
+  enabled=1
+  gpgcheck=1
+  repo_gpgcheck=1
+  gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
+          https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
+  EOF
+  setenforce 0
+  yum install -y docker kubelet kubeadm kubectl kubernetes-cni
+  systemctl enable docker && systemctl start docker
+  systemctl enable kubelet && systemctl start kubelet
+  ```
+
+The kubelet is now restarting every few seconds, as it waits in a crashloop for
+kubeadm to tell it what to do.
+
+Note: Disabling SELinux by running `setenforce 0` is required in order to allow
+containers to access the host filesystem, which is required by pod networks for
+example. You have to do this until kubelet can handle SELinux better.
```

Comment:
"You have to do this until kubelet can handle SELinux better" -> "This is a temporary workaround until future versions of Kubernetes improve compatibility between kubelet and SELinux."